<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_alm_ibp</class>
 <widget class="QDialog" name="Dlg_alm_ibp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>有创血压报警限设置</string>
  </property>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>中心静脉压:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_ibp1_shousuo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_ibp2_ss_h">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>90</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>收缩压</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_ibp1_pj_h">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>90</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>中心静脉压:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_ibp2_shuzhang">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>主动脉压:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_ibp1_pingjun">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>报警高限</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>中心静脉压:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_ibp1_sz_l">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>80</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>主动脉压:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>45</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>收缩压</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_ibp1_shuzhang">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>舒张压</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>报警</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_ibp2_pingjun">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>平均压</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_ibp1_sz_h">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>90</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_ibp2_pj_h">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>90</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_ibp2_sz_l">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>80</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>平均压</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_ibp1_pj_l">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>80</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>收缩压</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>45</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>主动脉压:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_ibp1_ss_l">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>80</number>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_ok">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_ibp2_ss_l">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>80</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_ibp1_ss_h">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>90</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>报警低限</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_ibp2_shousuo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_ibp2_sz_h">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>90</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_ibp2_pj_l">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>80</number>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_cancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
